File: 119.cpp
struct Solution {
    vector<int> getRow(int rowIndex) {
        vector<int> res(rowIndex + 1, 1);
        for (size_t i = 1; i <= rowIndex; ++i){
            res[i] = res[i - 1] * (rowIndex + 1 - i) / i;
        }
        return res;
    }
};

File: sol/119.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        res = [1] * (rowIndex + 1)
        for i in range(1, rowIndex + 1):
            res[i] = res[i - 1] * (rowIndex + 1 - i) // i
        return res
